### ISSUE TITLE
[Question] split into autoload?

### ISSUE THREAD

**Issue by monkoose** [tpope/vim-unimpaired#194]
vim-unimpaired such useful plugin for me, but at the same time i can use it a lot at some day, and none other day. Now it adds ~10ms startup time to vim, it is biggest number from all of my plugins (30+ plugins, almost noone from them take more then 1ms), and linewise unimpaired like the smallest plugin out there.
Could it be considered to split it?

**Comment by tpope** [comment]
Please show your work. I get a lot of performance issues that turn out to be from users not knowing how to read the profiler output.

**Comment by monkoose** [comment]
Sorry maybe language barrier, English not my native. Let me rephrase myself. I don't say that unimpaired slow in work, or that i have some performance issues with it, i said that it adds to vim startup time more then any other plugin i use (10ms still not so much, but what if all plugins will add that much?), because it is old plugin i believe pre autoload era, that you didn't refactor. I just think with refactoring and splitting plugin into autoload folder will decrease this startup time. Not sure if you have time for this, but if you interested i can make a PR.

**Comment by tpope** [comment]
Look at #148, where someone reorganized into an autoload file and it didn't improve startup time. I'm open to adding an autoload file if that can be fixed.
